vite.config.js

```diff
@@ -95,7 +95,6 @@ module.exports = ({ command, mode }) => {
               cert: fs.readFileSync("server.cert"),
             },
       watch: {
-        usePolling: true,
```

Comment:
I think I added this at some point to fix that vite was not detecting changes in my filesystem due to reasons beyond my comprehension. Is it affecting performance?

Reply:
Yes, I see `node` using 50% of cpu even when not doing any changes. Removing polling gets it back to normal

Reply:
Thanks for tracking this down, I'm having the CPU issue as well.